code/modules/projectiles/projectile/energy.dm

```diff
@@ -67,7 +67,7 @@
 /obj/item/projectile/energy/electrode/stunshot
 	nodamage = 0
 	damage = 10
-	agony = 80
+	agony = 50

```

Comment:
The stun egun uses the /obj/item/projectile/beam/stun
This is the projectile for the research acquired stun revolver and the stun shells for the shotguns